resources/RequestResetPassword.py

```diff
@@ -0,0 +1,46 @@
+from werkzeug.security import generate_password_hash, check_password_hash
+from flask_restful import Resource
+from flask import request
+from middleware.user_queries import user_check_email
+from middleware.reset_token_queries import add_reset_token
+import os
+import uuid
+import requests
+
+
+class RequestResetPassword(Resource):
+    def __init__(self, **kwargs):
+        self.psycopg2_connection = kwargs["psycopg2_connection"]
+
+    def get(self):
```

Comment:
@mbodeantor This endpoint is failing with the following error:

`415 Unsupported Media Type: Did not attempt to load JSON data because the request Content-Type was not 'application/json'.`

The `Content-Type` header is indeed set to `"application/json"`, so I'm not sure what's happening here.

<img width="613" alt="Screenshot 2024-02-26 at 9 36 15 AM" src="https://github.com/Police-Data-Accessibility-Project/data-sources-app/assets/68428039/5775fde5-3930-4f44-926d-b1e01171c82a">

Reply:
Update: hitting this from command line with the following, it works correctly. So I'm not sure what's different about this request from the browser compared with others?

```
curl -X GET -H "Content-Type: application/json" --data "{\"email\": \"mason.west38@ethereal.email\"}" http://localhost:5000/request-reset-password
```